Block/Adminhtml/System/Config/Field/MotoEnviroment.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2022 Adyen NV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Block\Adminhtml\System\Config\Field;
+
+class MotoEnviroment extends \Magento\Framework\View\Element\Html\Select
+{
+    /**
+     * Options
+     *
+     * @var array
+     */
+    protected $_options = [
+        '0' => 'Live',
+        '1' => 'Test'
```

Comment:
For consistency i would suggest using Adyen\Payment\Model\Config\Source\DemoMode for this

Reply:
`DemoMode` class extends `ArrayInterface` but not the `Select` which is part of `AbstractBlock` which necessary for the magento dynamic rows